packages/debugger-extension/src/debug-console-executor.ts

```diff
@@ -92,4 +96,25 @@ export class DebugConsoleCellExecutor implements IConsoleCellExecutor {
   }
 
   private _debuggerService: IDebugger;
+  private _trans: IRenderMime.TranslationBundle;
+}
+
+/**
+ * A namespace for DebugConsoleCellExecutor statics.
+ */
+export namespace DebugConsoleCellExecutor {
+  /**
+   * The instantiation options for a DebugConsoleCellExecutor.
+   */
+  export interface IOptions {
+    /**
+     * The debugger service for evaluating expressions in debug context.
+     */
+    debuggerService: IDebugger;
+
+    /**
+     * The translation bundle for internationalization.
+     */
+    trans: IRenderMime.TranslationBundle;
```

Comment:
Elsewhere we use `import { TranslationBundle } from '@jupyterlab/translation';` as the canonical import.